sentences/it/homeassistant_HassTurnOff.yaml

```diff
@@ -3,28 +3,25 @@ intents:
   HassTurnOff:
     data:
       - sentences:
-          - <turn_off> [<the>] <name>
+          - "<turn_off> [<the>] <name>"
       - sentences:
-          - <close> [<the>] <name>
-        response: cover
+          - "<close> [<the>] <name>"
+        response: "cover"
```

Comment:
in response it is quite unclear. But I might ommit `"` as you have defined `cover` and `cover_area`.
So we are not responding in string "cover" - this is not our answer. But we have defined some string named `cover` in program, so we are calling `cover` so it will be easyer to read the code if `"` is ommited.

Reply:
Can you please clarify?

Reply:
@auanasgheps

```
  - "<close> [<the>] <name>"
        response: cover 
```